<?xml version="1.0"?>
<launch>
	<node name="dsl_grid3d" pkg="dsl_gridsearch" type="dsl_grid3d_node" ns="spot" output="screen">
		<param name="use_gazebo_odom" type="bool" value="true"/>
                <param name="odom_topic" type="String" value="/cartographer/odom"/>
		<param name="odom_frame_id" type="String" value="map"/>
		<param name="map_topic" type="String" value="map"/>
		<param name="use_3d" type="bool" value="false"/>
		<param name="lower_thresh" type="int" value="75"/>
		<param name="upper_thresh" type="int" value="76"/>
		<param name="risk" type="int" value="2"/>

	</node>
</launch>
